<ui version="4.0" >
 <class>UIMessageBoxSettings</class>
 <widget class="QWidget" name="UIMessageBoxSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QGroupBox" name="gbActivateDock" >
     <property name="title" >
      <string>Activate dock on console start</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QLabel" name="lActivateDock" >
        <property name="text" >
         <string>Activate dock :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbActivateDock" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dbbButtons" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Help|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
